contracts/LosslessReporting.sol

```diff
@@ -298,4 +298,19 @@ contract LosslessReporting is ILssReporting, Initializable, ContextUpgradeable,
         emit CompensationRetrieve(_adr, _amount);
     }
 
-}
+    /// @notice This function allows the admin to retrieve tokens that are stuck in the contract
+    /// @param _token Address of the token to be retrieved
+    /// @param _to Address to which the tokens will be sent
+    /// @param _amount Amount of tokens to be retrieved
+    function retrieveRejectedReportTokens(ILERC20 _token, address _to, uint256 _amount) public onlyLosslessAdmin {
+        require(_to != address(0), "LSS: Cannot send to zero address");
+        require(_token != stakingToken, "LSS: Cannot retrieve staking tokens");
```

Comment:
The main goal of this function is to retrieve stuck staking tokens.

Reply:
Oops! Leftover from copy&paste [c9e7fd0](https://github.com/Lossless-Cash/lossless-v3/pull/116/commits/c9e7fd0810a1199ed1111f652221109326a01e32)